rules/network/zeek/zeek_dns_suspicious_zbit_flag.yml

```diff
@@ -37,6 +37,8 @@ detection:
     qtype_name:
       - 'NS'
       - 'ns'
+      - 'MX'
+      - 'MX'
```

Comment:
'MX' is in double. think you want to write 'mx'.

Reply:
yes, fixed.
thank you!